scaleway/resource_redis_cluster.go

```diff
@@ -151,7 +151,11 @@ func resourceScalewayRedisCluster() *schema.Resource {
 							Computed:    true,
 							Description: "UUID of the endpoint to be connected to the cluster",
 						},
-						"zone": zoneSchema(),
+						"zone": {
+							Type:        schema.TypeString,
+							Description: "Private network zone",
+							Computed:    true,
```

Comment:
Please add the description the documentation. This resource can change the zone?  Also do a helper, please.